src/main/java/ru/yandex/practicum/filmorate/model/activity/FriendPair.java

```diff
@@ -0,0 +1,66 @@
+package ru.yandex.practicum.filmorate.model.activity;
+
+import lombok.Getter;
+
+import javax.validation.constraints.NotNull;
+import java.util.Optional;
+
+/**
+ * класс хранения/передачи информации о статусе друзей
+ * хранит  информацию о двух id пользователей являющихся друзьями
+ * ТЗ-10
+ */
+public class FriendPair extends ActivityEvent {
+    @Getter
+    @NotNull(message = "user1 id is null")
+    private final Long friendIdOne;
+    @Getter
+    @NotNull(message = "user2 id is null")
+    private final Long friendIdTwo;
+
+    public FriendPair(Long idOne, Long idTwo) {
+        friendIdOne = idOne;
+        friendIdTwo = idTwo;
+    }
+
+    /**
+     * получение идентификатора "друга" по идентификатору пользователя
+     * идентификатор "друга" может быть в любого одного поля,
+     * если другое поле содержит идентификатор запрошенного пользователя
+     *
+     * @param userId идентификатор пользователя, для которого проводится "поиск" id друга
+     * @return Optional со значением идентификатора друга, если запрошенный пользователь содержится в экземпляре класса
+     * или Optional c null
+     */
+    @Override
+    public Optional<Long> getPairedId(Long userId) {
+        if (userId.longValue() == friendIdOne.longValue()) return Optional.of(friendIdTwo);
```

Comment:
Лучше выделять блок кода после условия с помощью фигурных скобочек `{}`. Так код выглядит более структурно и его проще читать
```suggestion
        if (userId.longValue() == friendIdOne.longValue()) {
            return Optional.of(friendIdTwo);
        }
```

Reply:
надо так надо :)